lib/rack/injectable.rb

```diff
@@ -17,7 +17,7 @@ def call(env)
     private
 
     def html?(headers)
-      headers['Content-Type'] =~ /html/
+      headers['Content-Type'].include? 'html'
```

Comment:
same  safe navigator issue as the other one
```suggestion
      headers['Content-Type']&.include? 'html'
```